projects/angular/storyteller/src/app/routes/feed/feed.component.ts

```diff
@@ -66,11 +68,9 @@ export class FeedComponent {
   // Filters stories by project id
   loadProjectStories(projectId: string): void {
     if (projectId !== 'all') {
-      this.easyStories = this.easyStories.filter(story => {
-        return story.project === projectId;
-      });
+      this.filteredEasyStories = this.storyService.getFilteredStories(this.easyStories, projectId);
```

Comment:
I decided, that that solution is simpler.
But if you wonder, how to load stories on each filter, I can show you an implementation.